lib/student.rb

```diff
@@ -1,11 +1,13 @@
+require_relative 'person'
 # Represents a student with a name, age, parent permission status, and classroom.
 # Inherits from Person class.
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :classroom
 
-  def initialize(name = 'Unknown', age = 0, parent_permission: true, classroom: 'Unknown')
-    super(name, age, parent_permission)
-    @classroom = classroom
+  def initialize(name = 'Unknown', age = 0, parent_permission: true, classroom: nil)
+    super(name, age, parent_permission: parent_permission)
+    @classroom = nil
+    self.classroom = classroom if classroom
```

Comment:
* Kindly add ` a method that serves as the 'belongs-to' side (a student belongs to a classroom)` 
* Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships
* [Check requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20belongs%2Dto%20side%20(a%20student%20belongs%20to%20a%20classroom).)